src/frontend/data/sideEffects/createThumbnail/createThumbnail.ts

```diff
@@ -0,0 +1,27 @@
+import { API_ENDPOINT } from '../../../settings';
+import { modelName } from '../../../types/models';
+import { Thumbnail } from '../../../types/tracks';
+
+/**
+ * Create a new timedtexttrack record for a language-mode combination.
+ * @param jwt The token that will be used to authenticate with the API.
+ * @param language The language for the new timedtexttrack (from the list of available choices).
+ * @param mode The mode for the new timedtexttrack.
+ */
+export const createThumbnail = async (jwt: string) => {
+  const response = await fetch(`${API_ENDPOINT}/${modelName.THUMBNAIL}/`, {
+    headers: {
+      Authorization: `Bearer ${jwt}`,
+      'Content-Type': 'application/json',
+    },
+    method: 'POST',
+  });
+
+  if (!response.ok) {
+    throw new Error('Failed to create a new thumbnail.');
```

Comment:
Should we have a dot at the end of an error message? True question.

Reply:
We have in all the front application.

Reply:
Not sure this is always the case, I mean some may miss.